docs/source/guide/guide-overview.rst

```diff
@@ -15,3 +15,8 @@ library with a long-term vision to be useful for quantum programmers using any q
 framework and any quantum backend. Today we support `Cirq <https://cirq.readthedocs.io/en/stable/>`_,
 `Qiskit <https://qiskit.org/>`_, and `PyQuil <https://pyquil-docs.rigetti.com/en/stable/>`_
 frontends and backends.
+
+The guide contains information of how :ref:`zero noise extrapolation <guide_zne>` can be implemented with the software and examples of how it can be applied to any simulator or quantum processor running :ref:`Back-end Plug-ins <guide-executors>`.
```

Comment:
My preference is to not include these because they are very apparent in the left/top nav and forward/back buttons. It's one less thing to have to update in the docs.

Reply:
Fine. Here I just updated the guide overview to reflect the new sections, it contained zne and executors since I added the research link as you suggested, which looked a bit awkward without information on the rest of the guide sections.